install.sh

```diff
@@ -200,6 +204,38 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+# Please sync with 'test-ruby()' in 'Library/Homebrew/utils/ruby.sh' from Homebrew/brew repository.
+test-ruby () {
+  [[ ! -x $1 ]] && { echo "false"; return 1; }
+  "$1" --enable-frozen-string-literal --disable=gems,did_you_mean,rubyopt -rrubygems -e \
+    "puts Gem::Version.new(RUBY_VERSION.to_s.dup).to_s.split('.').first(2) == \
+          Gem::Version.new('$required_ruby_version').to_s.split('.').first(2)" 2>/dev/null
+}
+
+usable_ruby() {
+  local ruby_exec
+  ruby_exec=$(which ruby)
+  test "$(test-ruby "$ruby_exec")" == true
```

Comment:
```suggestion
  test-ruby "$(command -v ruby)"
```

Reply:
I suggest merging `usable_ruby` and `test_ruby` into a single function. I forgot now… is it supposed to iterate through `PATH` and find the first usable Ruby anywhere in `PATH`? In case say Ruby 2.7 is at the head of `PATH`, but Ruby 2.6 is in there somewhere deeper.

Reply:
`which ruby` finds the first `ruby` in user's PATH. I can change it to mimick what we do in `utils/ruby.sh` to check all Ruby versions.

Reply:
Yeah, that's probably necessary.